src/DI/Loader/DoctrineAnnotationLoader.php

```diff
@@ -125,14 +125,14 @@ protected function analyseClass($class)
 	protected function acceptController(ClassType $class)
 	{
 		// Has class annotation @Controller?
-		if ($class->hasAnnotation('Controller')) return TRUE;
+		if ($class->hasAnnotation('Controller') || $class->hasAnnotation('API\Controller')) return TRUE;
```

Comment:
Since Controller is a doctrine annotations, ClassType::hasAnnotation should not be used for it at all. The annotation can be something completely different with `use ... as ...;`

Reply:
Yep. https://github.com/apitte/core/issues/6#issuecomment-354732810